### DISCUSSION TITLE
大佬们，求助docker驱动更改问题

### DISCUSSION THREAD

**Discussion by PeterHu3230** [ophub/amlogic-s9xxx-armbian#2380] [Amlogic] [answered]
今天刷了armbian，装好了docker，但是发现默认的驱动为cgroupv2，但我的服务依赖于v1版本的驱动，网上找了很多方法做了也无法更换驱动，看了一下armbian引导方式好像也不是grub，请问如何解决这个问题，谢谢！

**Comment by ophub** [comment]
看仓库的文档里有方法

**Reply by PeterHu3230** [reply]
感谢！但非常抱歉，我刚刚认真查阅了readme相关文档中的常见问题目录和正文，实在没有找到相关的信息，可否提供一下相关文档的定位，感谢！

**Reply by PeterHu3230** [reply]
![image](https://github.com/ophub/amlogic-s9xxx-armbian/assets/109458909/7838c3eb-5499-4c55-a60c-7e089d43463b)
我在dockerhub说明文档中找到了这个，但我发现armbian无法执行grubby命令，我也在对应目录下无法找到grub文件，同时geub-update命令也无法执行。

**Comment by ophub** [comment, marked as answer]
https://github.com/ophub/amlogic-s9xxx-armbian/blob/main/documents/README.cn.md

看12.14

**Reply by PeterHu3230** [reply]
非常感谢您，找到了！
